decapodes/configuration/decapodes.it

```diff
@@ -0,0 +1,42 @@
+struct Dimensionality
+	surface::Int64
+	embedding::Int64
+end
+
+@sum Region begin
+	Equation(equation::String)
```

Comment:
Is this intended as a region specified by a system of polynomials? We could do 

```julia
@sum Polynomial begin
  Var(x::Symbol)
  Mul(a::Number, p::Polynomial)
  Sum(ps::Vector{Polynomial})
  Prod(ps::Vector{Polynomial})
  Pow(p::Polynomial, n::UInt)
end

@sum Region begin
  AlgebraicSet(ps::Vector{Polynomial})
  Mesh(id::String, description::String, file::String, checksum::String, dimensionality::Dimensionality, vertex_count::Int64)
end
```

This would allow you to define a region of space as either the set of points that satisfy a polynomial equation (ex. the 2D sphere of radius r is `x^2 + y^2 - r^2 = 0`) or by an explicit mesh.